content/en/docs/collector/getting-started.md

```diff
@@ -154,7 +154,7 @@ $ sudo journalctl -u otel-collector
 
 ### Windows Packaging
 
-Every Collector release includes EXE and MSI packaging for Windows amd64 systems.
+Every Collector release includes EXE and MSI packaging for Linux amd64 systems.
```

Comment:
This looks wrong. EXE and MSI for Linux?

Reply:
I will close this PR and will re-submit once the source is fixed in this PR https://github.com/open-telemetry/opentelemetry-collector/pull/3356